packages/cubejs-api-gateway/src/dateParser.js

```diff
@@ -65,8 +65,7 @@ export function dateParser(dateString, timezone, now = new Date()) {
       moment.tz(timezone).endOf('day').add(1, 'day')
     ];
   } else if (dateString.match(/^from (.*) to (.*)$/)) {
-    // eslint-disable-next-line no-unused-vars,@typescript-eslint/no-unused-vars
-    const [all, from, to] = dateString.match(/^from (.*) to (.*)$/);
+    const [, from, to] = dateString.match(/^from\s{1,3}(.{0,50})\s{1,3}to\s{1,3}(.{0,50})$/);
```

Comment:
WDYT about using `.trim()` instead of `\s{1,3}`? Because it will 4 `\s` will pass from parent regexp.

<img width="1135" alt="image" src="https://github.com/user-attachments/assets/d81b7738-faef-4292-a406-4ddc1ceaa663" />

Reply:
Yeah! I like it! Simplifies even more :)